app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -103,47 +82,52 @@ export default {
     this.labelCategories = I18n.t("gobierto_data.projects.categories")
     this.labelshowAll = I18n.t("gobierto_data.projects.showAll")
     this.labelshowLess = I18n.t("gobierto_data.projects.showLess")
-    this.orderDatasets()
+    // TODO Datasets should be order in filter mixin
+    this.sortedItems = this.items.sort(({ attributes: { name: a } = {} }, { attributes: { name: b } = {} }) => a.localeCompare(b));
+
+    let { id } = this.$route.params
+    this.selectCurrentDataset(id)
+  },
+  watch: {
```

Comment:
Watch is recommend to be used when the there are dependencies between component attributes, i.e if the current dataset changes, what other properties change?